vercel.json

```diff
@@ -1,14 +1,173 @@
+      "source": "/posts/script-recorders/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/script-recorders/"
+    },
+    {
+      "source": "/posts/script-recorders/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/script-recorders/"
+    },
+    {
+      "source": "/posts/e2e-file-download/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-file-download/"
+    },
+    {
+      "source": "/posts/e2e-file-download/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-file-download/"
+    },
+    {
+      "source": "/posts/request-interception/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/request-interception/"
+    },
+    {
+      "source": "/posts/request-interception/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/request-interception/"
+    },
+    {
+      "source": "/posts/basics-scraping/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/basics-scraping/"
+    },
+    {
+      "source": "/posts/basics-scraping/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/basics-scraping/"
+    },
+    {
+      "source": "/posts/challenging-flows/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/challenging-flows/"
+    },
+    {
+      "source": "/posts/challenging-flows/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/challenging-flows/"
+    },
+    {
+      "source": "/posts/e2e-account-settings/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-account-settings/"
+    },
+    {
+      "source": "/posts/e2e-account-settings/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-account-settings/"
+    },
+    {
+      "source": "/posts/test-data-intro/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/test-data-intro/"
+    },
+    {
+      "source": "/posts/test-data-intro/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/test-data-intro/"
+    },
+    {
+      "source": "/posts/valuable-tests/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/valuable-tests/"
+    },
+    {
+      "source": "/posts/valuable-tests/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/valuable-tests/"
+    },
+    {
+      "source": "/posts/e2e-search/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-search/"
+    },
+    {
+      "source": "/posts/e2e-search/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-search/"
+    },
+    {
+      "source": "/posts/e2e-coupon/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-coupon/"
+    },
+    {
+      "source": "/posts/e2e-coupon/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-coupon/"
```

Comment:
What is the reason for having every post two times in this settings file?

Reply:
I made a script to generate this file, is probably an error with the script. Will fix that 👌

Reply:
👍